docs/t-sql/functions/char-transact-sql.md

```diff
@@ -193,6 +196,63 @@ first_byte_of_double_byte_character first_byte_of_double_byte_character
 NULL                                NULL                                         
 ```
   
+### G. Using CONVERT instead of CHAR to return legacy multibyte characters
+This example relies on the default codepage of the current database to map a two-byte legacy codepoint to a single character.

```

Comment:
what's a "legacy codepoint"?

Reply:
Thanks - that was unnecessarily pejorative. My assumption is that CONVERT isn't suitable for UTF-8 lookups, since NCHAR() is an option. It's the only option for SHIFT_JIS and company. I've changed the language to "decode" and "look up" to express a more subtle bias.